apps/core/tailwind.config.js

```diff
@@ -100,6 +100,7 @@ module.exports = {
                 subtitleSmallExtra: ['10px', '1'],
                 caption: ['12px', '1'],
                 captionSmall: ['11px', '1'],
+                captionSmallExtra: ['10px', '1'],
```

Comment:
Is this in the figma and aligned with the naming there as well?

Reply:
Yes! https://www.figma.com/file/xu2VnyPAuruNi74UQzGigW/02-Wallet?node-id=6461%3A95520&t=694zeFjQm2XxuhN0-0